ui/app/components/app/network-display/network-display.js

```diff
@@ -61,12 +68,17 @@ export default function NetworkDisplay({
       })}
       labelProps={{
         variant: TYPOGRAPHY.H7,
+        ...labelProps,
       }}
     />
   )
 }
 NetworkDisplay.propTypes = {
   colored: PropTypes.bool,
+  indicatorSize: PropTypes.oneOf(Object.values(SIZES)),
+  labelProps: PropTypes.shape({
+    ...Typography.propTypes,
```

Comment:
This should omit `children` and `className`, as `Chip.propTypes.labelProps` does. Perhaps this would be easiest?
```suggestion
    ...Chip.propTypes.labelProps,
```

Reply:
Yeah that's probably the better choice